inlong-sort/sort-connectors/jdbc/pom.xml

```diff
@@ -43,6 +43,11 @@
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
         </dependency>
+        <!--for oracle-->
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc8</artifactId>
+        </dependency>
```

Comment:
add <include>om.oracle.database.jdbc:ojdbc8</include>  in plugin

Reply:
On the premise that the user provides the oracle driver, this piece does not need to be done.

Reply:
ok